modules/create_timepoint/test/create_timepointTest.php

```diff
@@ -78,33 +77,6 @@ function testCreateTimepoint()
         );
     }
 
-    /**
-     * Create a timepoint with three parameters.
-     *
-     * @param string $canID      ID of candidate
-     * @param string $subproject text of subproject
-     * @param string $visitlabel text of visit label
-     *
-     * @return void
-     */
-    private function _createTimepoint($canID, $subproject, $visitlabel)
```

Comment:
This was a private method which wasn't called anywhere in the code (found when grepping for sleep)